mypy.ini

```diff
@@ -186,6 +187,8 @@ no_implicit_reexport=True
 ; Set this to skip until more of the codebase is typed
 [mypy-sentry.*]
 follow_imports = skip
+[mypy-bitfield.*]
+follow_imports = silent
```

Comment:
What's this for?

Reply:
otherwise it errors due to the `bitfield` import in `json.py`